trk/src/api/GymsApi.js

```diff
@@ -0,0 +1,22 @@
+import firestore from '@react-native-firebase/firestore';
+
+
+function GymsApi() {
+
+  async function fetchGyms() {
+    try {
+      const snapshot = await firestore().collection('gyms').get();
+      return snapshot.docs.map(doc => ({ label: doc.data().Name, value: doc.id }));
+    } catch (error) {
+      console.error("Error fetching gyms: ", error);
+      return [];
+    }
+  }
+
+  return {
+    fetchGyms,
+  }
+}
+
+export default GymsApi
+
```

Comment:
this function fetches all the gym documents in the gyms collection, maps each gym and sets the label as the name of the gym and the id as the value of each gym